src/test/java/com/cloudant/tests/util/Utils.java

```diff
@@ -33,4 +44,58 @@ public static String getHostName(String account) {
         }
 
     }
+
+    public static void removeReplicatorTestDoc(CloudantClient account, Response replicatorResponse)
+            throws Exception {
+
+        //Find replicator document for removal
+        ReplicatorDocument replicatorDoc = waitForReplicatorToComplete(account, replicatorResponse);
+
+        Response removeResponse = account.replicator()
+                .replicatorDocId(replicatorDoc.getId())
+                .replicatorDocRev(replicatorDoc.getRevision())
+                .remove();
+
+        assertThat(removeResponse.getError(), is(nullValue()));
+    }
+
+    public static ReplicatorDocument waitForReplicatorToComplete(CloudantClient account, Response
```

Comment:
I actually meant to swap `replicatorResponse` for just the doc ID, I think leaving the account as being passed in is fine. For some reason I missed the `account` param on first read.